src/main/resources/assets/computercraft/lua/rom/apis/keys.lua

```diff
@@ -53,6 +53,7 @@ for nKey, sKey in pairs( tKeys ) do
 	keys[sKey] = nKey
 end
 keys["return"] = keys.enter
+keys.scollLock = keys.scrollLock
```

Comment:
You might as well change this to `keys["scrollLock"] to be consistent with the line above.

Reply:
I believe it's only ["return"]` because that is a Lua keyword, and so `keys.return` is invalid syntax.